fastlane/Snapfile

```diff
@@ -0,0 +1,31 @@
+# Uncomment the lines below you want to change by removing the # in the beginning
+
+# A list of devices you want to take the screenshots from
+devices([
+  "iPhone 15 Pro Max",
+  "iPhone 14 Plus",
+  "iPhone 8 Plus",
+])
+
+languages([
+  "en-US",
+])
+
+# The name of the scheme which contains the UI Tests
+scheme("PAWS")
+
+# Where should the resulting screenshots be stored?
+output_directory("./screenshots")
```

Comment:
I would use dot prefixed folder names for temp storage, we do the same thing for .derivedData in the other fastlane actions.
```suggestion
output_directory("./.screenshots")
```